v2/pages/issuer/opportunities/create.jsx

```diff
@@ -0,0 +1,152 @@
+import { useInput } from '../../../hooks';
+import { colors } from '../../../assets/styles/constants';
+import { Heading, FormGroup, Button } from '../../../components/UI';
+import GoBack from '../../../components/goBack';
+
+const DOMAINS = [
+  'Digitale geletterdheid',
+  'Duurzaamheid',
+  'Ondernemingszin',
+  'Onderzoek',
+  'Wereldburgerschap'
+];
```

Comment:
I think we're going to need this across other components such as badge-related ones and even when updating opportunities, so I wouldn't mind putting this in the constants file